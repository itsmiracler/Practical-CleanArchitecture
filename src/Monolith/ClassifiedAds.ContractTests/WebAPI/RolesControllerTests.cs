using ApprovalTests;
using ApprovalTests.Reporters;
using ClassifiedAds.WebAPI.Models.Roles;
using NJsonSchema;
using Xunit;

namespace ClassifiedAds.ContractTests.WebAPI
{
    [UseReporter(typeof(VisualStudioReporter))]
    public class RolesControllerTests
    {
        [Fact]
        public void RoleModelTest()
        {
            var schema = JsonSchema.FromType<RoleModel>().ToJson();
            Approvals.Verify(schema);
        }
    }
}
