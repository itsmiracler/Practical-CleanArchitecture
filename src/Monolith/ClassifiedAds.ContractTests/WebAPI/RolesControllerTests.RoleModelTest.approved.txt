{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "title": "RoleModel",
  "type": "object",
  "additionalProperties": false,
  "properties": {
    "Id": {
      "type": "string",
      "format": "guid"
    },
    "Name": {
      "type": [
        "null",
        "string"
      ]
    },
    "NormalizedName": {
      "type": [
        "null",
        "string"
      ]
    },
    "ConcurrencyStamp": {
      "type": [
        "null",
        "string"
      ]
    }
  }
}